config/custom_items.txt

```diff
@@ -890,3 +890,9 @@ ckey: zodiacshadow
 character_name: Nehi Maximus
 item_path: /obj/item/device/radio/headset/fluff/zodiacshadow
 }
+
+{
+ckey: jacknoir413
+character_name: Areax Third
+item_path: /obj/item/weapon/storage/box/fluff/stunstaff
```

Comment:
The reason given for accepting the application is that it's functionally identical to a baton, so it should probably be restricted to jobs that are normally allowed a baton.

Reply:
Done, slipped my mind